src/main/java/org/jitsi/jigasi/xmpp/CallControlMucActivator.java

```diff
@@ -268,15 +272,13 @@ public void registrationStateChanged(RegistrationStateChangeEvent evt)
      */
     private void joinCommonRoom(ProtocolProviderService pps)
     {
-        OperationSetMultiUserChat muc
-            = pps.getOperationSet(OperationSetMultiUserChat.class);
-
         String roomName = pps.getAccountID()
             .getAccountPropertyString(ROOM_NAME_ACCOUNT_PROP);
 
         if (roomName == null)
         {
-            logger.warn("No brewery name specified for:" + pps);
+            // this is the common account for connecting which is configured
```

Comment:
How do we know that this is the account which is not supposed to have a brewery? Wouldn't this happen if one of the other accounts in misconfigured, in which case it is helpful to have the log message? Why does the comment mention authorization, is it related to this at all?

Reply:
Hum, wrong comment, will fix it :( 
So this is a RegistrationStateChangeListener which we add to the protocol provider ... hum, I will further check it ... I was seeing this log "No brewery name specified for:" multiple times in production ... and wanted to remove it as it is not relevant ... but you make me think that something else is happening and I will check it ...